### ISSUE TITLE
WatchOS library converting fails

### ISSUE THREAD

**Issue by mykyta-bondarenko-gl** [mono/Embeddinator-4000#657]
### Steps to Reproduce

1. Create `watchOS` library
2. Build it
3. Try to convert it like an [iOS library](https://docs.microsoft.com/en-us/xamarin/tools/dotnet-embedding/get-started/objective-c/ios)
Use next command:
```
objcgen [path_to_the_dll]/test-watch.dll --target=framework --platform=watchOS --outdir=output -c --debug
```

### Expected Behavior
The library is converted successfully

### Actual Behavior
Converting fails with `error MT0077: WatchOS projects must be extensions.`

### Environment

1. Library
Embeddinator-4000 v0.1 (0.4: 7664197)
2. VSfM
https://gist.github.com/mykyta-bondarenko-gl/677d0202be21d1bcd62bc4628f981940

### Build Logs

1. Output
https://gist.github.com/mykyta-bondarenko-gl/f48f92d9eb01bd1043630aa6e572468c
2. Generated folder
[Output.zip](https://github.com/mono/Embeddinator-4000/files/1941771/output.zip)

**Comment by chamons** [comment]
I believe known @rolfbjarne but I didn't see an existing issue?

**Comment by rolfbjarne** [comment]
Yes, this is a known issue (it should be noted in the release notes).

**Comment by fotiDim** [comment]
Any progress on that?
